css/docs.css

```diff
@@ -0,0 +1,254 @@
+@import url('https://fonts.googleapis.com/css2?family=Hind:wght@300&display=swap');
+
+
+
+/* @media (min-width: 811px) {
+}
+ */
+
+.doc-cont{
+    width: 100%;
+    height: 87vh;
+    background-color: #fff;
+    display: flex;
+}
+
+.doc-sidebar{
+    width: 20%;
```

Comment:
maybe try 13%-16% which ever looks good enough

Reply:
Sure I will do it.